mslib/mswms/mss_plot_driver.py

```diff
@@ -274,7 +274,7 @@ def update_plot_parameters(self, plot_object=None, figsize=None, style=None,
         figsize = figsize if figsize is not None else self.figsize
         noframe = noframe if noframe is not None else self.noframe
         init_time = init_time if init_time is not None else self.init_time
-        valid_time = valid_time if valid_time is not None else self.valid_time
```

Comment:
I propose to consistently use in the server "valid_time" instead of fc_time. Maybe in a separate issue.

Reply:
I agree. I am just reacting to errors here, since self.valid_time actually doesn't exist.